src/models/OrgSettings.ts

```diff
@@ -0,0 +1,23 @@
+export default interface OrgSettings {
+  status: ServiceStatus;
+  emergenciesEnabled: boolean;
+  messageOfTheDay: MessageOfTheDay;
+}
+
+export interface MessageOfTheDay {
+  message: string;
+  dateRange: DateRange;
+}
+
+export interface DateRange {
+  startDate: string;
+  endDate: string;
+}
+
+export enum ServiceStatus {
```

Comment:
@charliefoxtwo I've put placeholders in the enum, what should each one be named to be consistent with what the API expects?

Reply:
iirc, should be:
```
UNKNOWN,
HEALTHY,
SLIGHTLY_DEGRADED,
HEAVILY_DEGRADED,
OFFLINE,
```